internal/telemetry/client.go

```diff
@@ -341,10 +382,7 @@ func (c *Client) flush() {
 
 	go func() {
 		for _, r := range submissions {
-			err := c.submit(r)
-			if err != nil {
-				c.log("telemetry submission failed: %s", err)
-			}
+			r.submit()
```

Comment:
We're ignoring this error now? Why remove the code we had before?

Reply:
I originally intended to move all the error logging inside the submit method which is why i removed the code, but just realized there are some errors that are returned and not logged so I will change the code back!